backend/src/main/java/com/bakdata/conquery/models/query/concept/TableExportQuery.java

```diff
@@ -57,45 +71,40 @@
 	@Valid
 	private List<CQUnfilteredTable> tables;
 
+	@InternalOnly
+	@JsonDeserialize(keyUsing = NsIdReferenceKeyDeserializer.class)
+	@JsonSerialize(keyUsing = IdReferenceKeySerializer.class)
+	private Map<Column, Integer> positions;
```

Comment:
Gibt es bei Maps nicht auch so etwas wie 
```java
private Map< @NsIdRef Column, Integer> positions;
```
 ?

Reply:
Ich weiß nicht ob das NsIdRef das kann, schau es mir mal an.

Reply:
Leider nein, die müssen in einem bestimmten Format schreiben/lesen